packages/assets/README.md

```diff
@@ -1,16 +1,22 @@
 # VA Mobile Design System - Assets Package
 
-The Assets package contains static assets such as fonts and SVG icons for use by VA mobile applications. 
+The Assets package contains static assets such as fonts, images, and SVG icons for use by VA mobile applications. 
 
 ## For consumers
 
-The Assets package is a peer dependency to the Components package so it is expected to be added separately via your package manager (e.g. yarn) alongside using components. It does not contain any code, so it is up to the consumer to configure the package to be leveraged by their app, [notably the fonts so they are available within your app](https://blog.logrocket.com/adding-custom-fonts-react-native/) for the components.
+The Assets package is a peer dependency to the Components package so it is expected to be added separately via your package manager (e.g. yarn) alongside using components. Aside from internal scripts, it does not contain any code, so it is up to the consumer to configure the package to be leveraged by their app, [notably the fonts so they are available within your app](https://blog.logrocket.com/adding-custom-fonts-react-native/) for the components.
```

Comment:
In retrospect, instead of changing this line we should probably just add an `.npmignore` file like components/tokens have and remove the SVG script from the published package. Then it'd still be accurate for consumers to say it doesn't contain code.